tools/xyz2pkf.cpp

```diff
@@ -7,7 +7,7 @@
 
 #ifndef DOXYGEN_SHOULD_SKIP_THIS
 
-#include "../include/Curve.h"
+#include <Curve.h>
```

Comment:
Just curious, why the change away from relative here?

Reply:
No particularly great reason, but it seems to me you should either use `-I../include` or `"../include/..."`, but not both...